src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
Stylesheets with a `.scss` extension cannot be linked directly in HTML. They must be compiled to CSS. Change the `href` attribute to point to a `.css` file.